lib/stellar/client.rb

```diff
@@ -46,11 +46,11 @@ def friendbot(account)
       raise NotImplementedError
     end
 
-    # Contract Stellar::Account => Stellar::AccountInfo
-    Contract Stellar::Account => Any
+    Contract Stellar::Account => Stellar::AccountInfo
     def account_info(account)
       account_id  = account.address
-      @horizon.account(account_id:account_id)
+      resource = @horizon.account(account_id:account_id)._get
```

Comment:
Instead of introducing `AccountInfo`, how about we just return the result of `#_get` and call it a day?

Reply:
I was gonna do that first, but I saw a commented out `Contract Stellar::Account => Stellar::AccountInfo` :), but I could get remove that.

Reply:
:)  Yeah, I think I was getting ahead of myself with the comment line.  Please remove the type, and we'll add it back when we specifically need it. 

Thanks!